test/hotspot/jtreg/compiler/jvmci/compilerToVM/ConstantPoolTestsHelper.java

```diff
@@ -97,6 +98,25 @@ public int getCPCacheIndex(int cpi) {
         }
     }
 
+    /**
+     * Select an arbitrary bytecode of the type associated with the Constant pool entry type
+     *
+     * @param cpType Constant type from the Constant pool cache
+     * @return Arbitrary opcode of the kind associated with the CP entry
+     */
+    public static int getDummyOpcode(ConstantTypes cpType) {
+        switch (cpType) {
+            case CONSTANT_FIELDREF:
+              return Bytecodes.GETFIELD;
+          case CONSTANT_METHODREF:
+          case CONSTANT_INTERFACEMETHODREF:
+              return Bytecodes.INVOKEVIRTUAL;
```

Comment:
It's better to return INVOKEINTERFACE for CONSTANT_METHODREF.

Reply:
Why?  It seems like it should be

`CONSTANT_METHODREF: should return Bytecodes.INVOKEVIRTUAL;
 CONSTANT_INTERFACEMETHODREF: should return Bytecodes.INVOKEVIRTUAL`